src/org/testKitGen/TestInfoParser.java

```diff
@@ -147,12 +147,14 @@ public TestInfo parse() {
 		getElements(ti.getPlatformRequirementsList(), "platformRequirementsList", "platformRequirements", null, ti.getTestCaseName());
 
 		List<String> variations = new ArrayList<String>();
-		getElements(variations, "variations", "variation", null, ti.getTestCaseName());
 		List<Variation> listOfVars = new ArrayList<Variation>();
-		for (int i = 0; i < variations.size(); i++) {
-			String subTestName = ti.getTestCaseName() + "_" + i;
-			Variation var = parseVariation(subTestName, variations.get(i), ti.getPlatform(), ti.getPlatformRequirementsList());
-			listOfVars.add(var);
+		if (System.getenv("JVM_OPTIONS") == null) {
```

Comment:
Will this be obvious to the suer that this rule is being applied?

Reply:
Maybe I can print something to stdout.

Reply:
Added warning message.